api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -354,6 +354,29 @@ message AssistCompletionEvent {
   int64 completion_tokens = 4;
 }
 
+// AssistExecutionEvent is an event that is emitted when an Assist command
+// execution is triggered by the user.
+message AssistExecutionEvent {
+  // ConversationId is the UUID that identifies a single Assist conversation
+  string conversation_id = 1;
+  // NodeCount is the number of nodes the command was executed on
+  int64 node_count = 2;
+  // TotalTokens is the total amount of token used to satisfy this request
+  int64 total_tokens = 3;
```

Comment:
I don't think this is needed. It's always the sum of prompt + competition. 
Do you know if we can add fields in posthog when building a query?

Reply:
I found it very hard to do any sort of computations in Prehog, except for what is pre-defined (e.g. grouping by "User ID" etc.). There's [HogQL](https://posthog.com/docs/hogql), but apparently it is unavailable in our version, or something